install/kubernetes/istio.yaml

```diff
@@ -622,8 +651,8 @@ spec:
   ports:
   - port: 8080
     name: http-discovery
-  - port: 8081
-    name: http-apiserver
+  - port: 443
```

Comment:
Is this the only change needed to enable webhooks?

Reply:
No, users need to run install/kubernetes/external-admission-webhook-gke-1-7-x-workaround.sh. This concerts the istio-pilot service to type=LoadBalancer, generates certs, and plumbs them through to k8s secret. The secret creation triggers the admission webhook to self-register.

Reply:
We will need to document the extra step somewhere on istio.io.